src/components/Calender.vue

```diff
@@ -118,7 +123,6 @@
 
 		padding: 0.22em 0.11em 0;
 		margin: 0;
-		margin-bottom: auto;
```

Comment:
In flex layout, `auto` margin takes up all available space. You should at least add it back when the titles disappear to keep the usernames from clustering up at the top:

<img width="1403" alt="Screen Shot 2022-10-22 at 12 41 47 PM" src="https://user-images.githubusercontent.com/19389660/197359544-c5eded74-d16f-4535-a2b6-94a8999fea09.png">